src/librustc_typeck/diagnostics.rs

```diff
@@ -1972,6 +1972,88 @@ To learn more about traits, take a look at the Book:
 https://doc.rust-lang.org/book/traits.html
 "##,
 
+E0174: r##"
+This error occurs because of the explicit use of unboxed closure methods
+that is an experimental feature in current Rust version.
+
+Example of erroneous code:
+
+```compile_fail
+fn foo<F: Fn(&str)>(mut f: F) {
+    f.call(("call",));
+    // error: explicit use of unboxed closure method `call`
+    f.call_mut(("call_mut",));
+    // error: explicit use of unboxed closure method `call_mut`
+    f.call_once(("call_once",));
+    // error: explicit use of unboxed closure method `call_once`
+}
+
+fn bar(name: &str) {
+    println!("Calling {} it works!", name);
+}
+
+fn main() {
+    foo(bar);
+}
+```
+
+Rust’s implementation of closures is a bit different than other languages.
+They are effectively syntax sugar for traits `Fn`, `FnMut` and `FnOnce`.
+To understand better how the closures are implemented see here:
+https://doc.rust-lang.org/book/closures.html#closure-implementation
+
+To fix it you can just call them using `()`, like `foo()`. When you execute
```

Comment:
"just" should probably not be used in documentation or error messages. It doesn't add anything, and can seem negative.

I would also maybe slightly re-phrase: "To fix this you can call them using parenthesis, like this: `foo()`.